recipes-graphics/imx-gpu-viv/imx-gpu-viv-6.inc

```diff
@@ -131,8 +133,11 @@ PACKAGE_FP_TYPE = "hardfp"
 HAS_GBM     = "false"
 HAS_GBM_mx8 = "true"
 
-GLES3_HEADER_REMOVALS       = "gl31.h gl32.h"
-GLES3_HEADER_REMOVALS_mx8mq = "gl32.h"
+# GLES 3.1 is supported for i.MX 8 Series only
+# GLES 3.2 is supported for i.MX 8 Family only
```

Comment:
The commit log is confusing. 

I think you could instead set it for each SoC as it would be clearer and easier to understand.